homeassistant/components/playstation_network/helpers.py

```diff
@@ -92,10 +92,7 @@ async def get_data(self) -> PlaystationNetworkData:
         data.username = self.user.online_id
         data.account_id = self.user.account_id
 
-        data.available = (
-            data.presence.get("basicPresence", {}).get("availability")
-            == "availableToPlay"
-        )
+        data.available = data.presence["basicPresence"]["availability"]
```

Comment:
```suggestion
        data.available = data.presence.get("basicPresence", {}).get("availability")
```
Are the keys always present (even for legacy profile)? Since it was using `get` previously, I wonder if we should keep `get`.

Reply:
Since `data.presence["basicPresence"]` was also already used a few lines bellow, it should be safe.